mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/MongoDBDialect.java

```diff
@@ -25,6 +25,8 @@
 import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
 
+import org.apache.commons.lang3.StringEscapeUtils;
```

Comment:
OK, now that both of you have converged, I would vote for suppressing the dependency to commons-lang3. It's a big dependency for a single method.

We should be able to reimplement it.